### ISSUE TITLE
[vue3] click-events-have-key-events raised on button elements

### ISSUE THREAD

**Issue by schalkneethling** [dequelabs/axe-core#3549]
Product: axe Linter in VSCode

Expectation:

I would not expect a `click-events-have-key-events` error raised on `button` elements.

Actual:

In VSCode I get:

```
Axe Linter (click-events-have-key-events): Ensure a clickable non-interactive element has at least one keyboard event listener
```

This is being shown for an `@click` handler added to a `button` element in Vue3.

Motivation:

I believe that `button` elements are interactive elements(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) and therefore this error should not be raised.

<pre><code>
extension version:
Name: axe Accessibility Linter
Id: deque-systems.vscode-axe-linter
Description: Accessibility linting for HTML, Angular, React, Markdown, and Vue
Version: 4.4.0
Publisher: Deque Systems
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter
</code></pre>

**Comment by WilcoFiers** [comment]
@schalkneethling Thanks for bringing this up. This is something that should have already been fixed in 4.4.0. We had some people experience issues i upgrading. Would you mind installing 4.4.1 and letting us know if this addressed the issue? This came out yesterday.

**Comment by schalkneethling** [comment]
> @schalkneethling Thanks for bringing this up. This is something that should have already been fixed in 4.4.0. We had some people experience issues i upgrading. Would you mind installing 4.4.1 and letting us know if this addressed the issue? This came out yesterday.

Thanks so much. Will let you know as soon as I upgraded.

**Comment by schalkneethling** [comment]
@WilcoFiers, I have updated to 4.4.1, and the problem seems fixed. 🎉  Thanks!
